ansible/roles/unit-tests/tasks/docker.yml

```diff
@@ -0,0 +1,63 @@
+---
+- name: Start test-runner
+  community.docker.docker_container:
+    name: test-runner
+    image: quay.io/centos/centos:stream9
+    interactive: true
+    cgroupns_mode: host
+    pid_mode: host
+    privileged: true
+  register:
+    test_result
+
+- name: Install dependencies
+  community.docker.docker_container_exec:
+    container: test-runner
+    command: >
+      dnf install --enablerepo=crb -y
```

Comment:
we might be able to reduce duplication between here and `podmand.yml` by having a Containerfile for this image. I don't think we'd need to build it locally and publish it though -- perhaps we can use `docker_image_build` and `podman_image` to build it on the VM to follow this existing behaviour